google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -87,6 +87,17 @@ public static Mutation fromProtoUnsafe(List<com.google.bigtable.v2.Mutation> pro
     return mutation;
   }
 
+  /**
+   * Wraps the List of protobuf {@link com.google.bigtable.v2.Mutation}. This method creates a
+   * mutation instance that is idempotent in nature, which could safely be retried.
```

Comment:
This javadoc is not quite right. Please rephrase to:

```
Constructs a row mutation from an existing protobuf object.

<p>Callers must ensure that the protobuf argument is not using serverside timestamps. The client assumes that all mutations are idempotent and will retry in case of transient errors. This can lead to row duplication.

<p>When applied, the resulting Mutation object will ignore the project id and instance id in the table_name and instead apply the configuration in the client
```

Reply:
Actually nevermind, please remove this fromProto method altogether. This object is not a top level model so from proto is not needed here

Reply:
It seems we need this utility for `RowMutation#fromProto`. At present, there is no other option to create `model.Mutation` from List of proto `com.google.bigtable.v2.Mutation`.

Reply:
ok, can it be package private? or internalapi?